### PR TITLE
Fix configmap test under some OpenShift environments

### PR DESCRIPTION
**Pull request by juangon**

### CODE REVIEW

File: configmap/src/test/java/io/thorntail/openshift/ts/configmap/ConfigMapIT.java [hunk 1]
```diff
@@ -71,12 +73,14 @@ public void _3_updateConfigMap() throws Exception {
...
-        .when()
-                .get()
-        .then()
-                .statusCode(500);
+        await().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
+            given()
+                    .baseUri(greetingUrl.toString())
+             .when()
+                    .get()
+             .then()
```
Comment (Ladicek) [inline configmap/src/test/java/io/thorntail/openshift/ts/configmap/ConfigMapIT.java]: Wrong indentation on `.when` and `.then`.

**Comment by Ladicek** [comment]
Yea this makes sense to me. What we've seen is 404, which is legitimate and awaiting 500 looks reasonable to me.

**Comment by juangon** [comment]
@Ladicek pushed changes..

**Comment by Ladicek** [comment]
Thanks!

**Pull request merged**